src/Essentials/src/Screenshot/Screenshot.ios.cs

```diff
@@ -17,26 +17,13 @@ partial class ScreenshotImplementation : IPlatformScreenshot, IScreenshot
 		public bool IsCaptureSupported =>
 			true;
 
-		[System.Runtime.Versioning.SupportedOSPlatform("ios13.0")]
-		[System.Runtime.Versioning.SupportedOSPlatform("tvos13.0")]
 		public Task<IScreenshotResult> CaptureAsync()
 		{
-			var scenes = UIApplication.SharedApplication.ConnectedScenes;
-			//#pragma warning disable CA1416 // Known false positive with Lambda expression
-			var currentScene = scenes.ToArray().Where(n => n.ActivationState == UISceneActivationState.ForegroundActive).FirstOrDefault();
-			//#pragma warning restore CA1416
-			if (currentScene == null)
-				throw new InvalidOperationException("Unable to find current scene.");
-
-			var uiWindowScene = currentScene as UIWindowScene;
-			if (uiWindowScene == null)
-				throw new InvalidOperationException("Unable to find current window scene.");
-
-			var currentWindow = uiWindowScene.Windows.FirstOrDefault(n => n.IsKeyWindow);
+			var currentWindow = WindowStateManager.Default.GetCurrentUIWindow();
 			if (currentWindow == null)
 				throw new InvalidOperationException("Unable to find current window.");
 
-			return CaptureAsync(currentWindow.Layer, true);
+			return CaptureAsync(currentWindow);
```

Comment:
This was actually incorrect because this code says capture just the layer and skip the children - so you end up with just a solid image of background color.